sync_packages.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+robot_name="${HOSTNAME//-b1}"
+
+if [ "$HOSTNAME" != "$robot_name-b1" ]; then 
+	echo "FATAL: CAN ONLY BE EXECUTED ON BASE PC"
+	exit
+fi
+
+sudo apt list --installed > /tmp/package_list
+perl -pi -e 's{ / .*? ]}{}x' /tmp/package_list
+sed -i 's/Listing...//' /tmp/package_list 
+sed -i ':a;N;$!ba;s/\n/ /g' /tmp/package_list
+tr '\n' ' ' < /tmp/package_list | sed -i '$s/ $/\n/' /tmp/package_list
+packages=$(cat /tmp/package_list)
+
+pcs="
```

Comment:
have you considered storing this list somewhere in /etc?
This way it might be easier to setup robots with different number of PCs